tests/storage/test_room_search.py

```diff
@@ -72,3 +75,110 @@ def test_null_byte(self):
         )
         if isinstance(store.database_engine, PostgresEngine):
             self.assertIn("alice", result.get("highlights"))
+
+    @skip_unless(not USE_POSTGRES_FOR_TESTS, "requires sqlite")
```

Comment:
Isn't this test valid on both postgres and sqlite?

Reply:
Yes, let's run it on both.